lib/utils/write-to-clipboard.ts

```diff
@@ -0,0 +1,3 @@
+export async function writeToClipboard(url: string) {
```

Comment:
The function no longer needs to be async as it's the `setTimeout` callback that handles the async function call now. The `await writeToClipboard(shortUrl);` can be `writeToClipboard(shortUrl);` now.

```suggestion
export function writeToClipboard(url: string) {
```